src/qgis_geonode/gui/layer_properties_config_widget.py

```diff
@@ -20,7 +20,8 @@ def createWidget(self, layer, canvas, dock_widget, parent):
         return LayerPropertiesConfigWidget(layer, canvas, parent)
 
     def supportsLayer(self, layer):
-        return True
+        return (layer.type() == QgsMapLayerType.VectorLayer) | \
+               (layer.type() == QgsMapLayerType.RasterLayer)
```

Comment:
This seems more idiomatic:

```suggestion
        return layer.type() in (QgsMapLayerType.VectorLayer, QgsMapLayerType.RasterLayer)
```